src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/support/SupportController.java

```diff
@@ -154,12 +161,15 @@ private void resendStaffNotificationOnMoreTimeRequested(Claim claim) {
         moreTimeRequestedStaffNotificationHandler.sendNotifications(event);
     }
 
-    private void resendStaffNotificationOnDefendantResponseSubmitted(Claim claim) {
+    private void resendStaffNotificationOnDefendantResponseSubmitted(Claim claim, String authorization) {
         if (!claim.getResponse().isPresent()) {
-            throw new ConflictException(CLAIM + claim.getId() + " does not have associated response");
+            throw new ConflictException(CLAIM + claim.getReferenceNumber() + " does not have associated response");
         }
-        DefendantResponseEvent event = new DefendantResponseEvent(claim);
+        DefendantResponseEvent event = new DefendantResponseEvent(claim, authorization);
         defendantResponseStaffNotificationHandler.onDefendantResponseSubmitted(event);
+        if (!claim.getClaimDocument(DEFENDANT_RESPONSE_RECEIPT).isPresent()) {
```

Comment:
I don't think we need this here, please remove this block. We are already generating and uploading where we are trying to get the document from doc-store.

Reply:
This is meant for the support team. I have put this in for now, as safety net because we don't yet have a support controller for DM. I will be refactoring this code soon.

Reply:
Please remove this unnecessary "safety net"

Reply:
this is not requirement, please stick to story requirements